src/components/checkbox/checkbox-validations.stories.mdx

```diff
@@ -2,9 +2,20 @@ import { Meta, Story, Canvas } from "@storybook/addon-docs";
 import * as stories from "./checkbox-validations.stories";
 
 import { Checkbox, CheckboxGroup } from ".";
+import Link from "../link";
 
 <Meta title="Checkbox/Test" parameters={{ info: { disable: true } }} />
 
+# Checkbox
+
+  <Link
```

Comment:
Can I just check why here we are using the Link component (and a couple of other places) but elsewhere we're using HTML Anchor tags?

Reply:
anchor tags only :)